src/ResourceManager/MarketplaceOrdering/AzureRM.MarketplaceOrdering.psd1

```diff
@@ -0,0 +1,126 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.MarketplaceOrdering'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 9/14/2017
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
```

Comment:
@prbansa please set this to version `0.1.0`, which is the initial version we set for all modules.